source/ocsp-support/ocsp-support.rst

```diff
@@ -3,14 +3,14 @@ OCSP Support
 ============
 
 :Spec Title: OCSP Support
-:Spec Version: 1.0.2
+:Spec Version: 1.3.0
```

Comment:
Adding a URI option seeems worthy a  minor version bump. (1.2.0 was for the cache)